test/gen_integration.sh

```diff
@@ -287,7 +287,21 @@ languageVariables:
           echo 'Curling service IP'
           curl -m 3 \$SERVICEIP:$serviceport
           kill \$tunnelPID
-      - run: ./draft -b main -v generate-workflow -d ./langtest/ -c someAksCluster -r someRegistry -g someResourceGroup --container-name someContainer
+      - run: ./draft -b main -v generate-workflow -d ./langtest/ -c someAksCluster -r someRegistry -g someResourceGroup --container-name someContainer --deploy-type helm
+      # Validate generated workflow yaml
```

Comment:
lgtm just need to run this gen_integration to update the workflow and it's good to go

Reply:
I've added the integration test for update in the other PR.